rqt_multiplot/CMakeLists.txt

```diff
@@ -318,9 +318,6 @@ install(TARGETS rqt_multiplot
 )
 
 install(PROGRAMS scripts/rqt_multiplot
-  DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
```

Comment:
Why is this not required? This file and the `setup.py` file are similar to the `rqt_image_view`. Did this change over the time, because in `rqt_common_plugins` it's about 50:50, some packages use the same 'setup' as you propose and some the same as this package.